Modules/Core/Mesh/include/itkTriangleMeshCurvatureCalculator.h

```diff
@@ -31,6 +31,7 @@
 
 #include <iostream>
 #include "itkMesh.h"
+#include "itkTriangleCell.h"
```

Comment:
When you have an "**and**" in commit message, it is a good indication it should be split into two commit. There is no limit how small commits can be. I do realize this is nitpicking, so feel free to ignore this time.